internal/server/v2/facet/golden/contained_in_facet/IND_AA1.json

```diff
@@ -39,11 +36,6 @@
       "measurement_method": "OECDRegionalStatistics",
       "observation_period": "P1Y"
     },
-    "1456184638": {
```

Comment:
Latest population data in KG is from 2023 July, seems like was dropped in refresh